connection-limit/connection_limit_kern.c

```diff
@@ -327,4 +339,4 @@ int _xdp_limit_conn(struct xdp_md *ctx)
     return XDP_PASS;
 }
 
-char _license[] SEC("license") = "GPL";
+char _license[] SEC("license") = "Dual BSD/GPL";
```

Comment:
According to git, Santhosh was the author of this file, so only Santhosh can change the license.  Hence we need Santhosh to approve this PR before merging it.  This does seem unrelated to "fixing L4 offset for IP packets with Options field" so I would recommend all license related changes in this PR be moved to a separate PR.